Libplanet.Net.Tests/Messages/MessageValidatorTest.cs

```diff
@@ -124,19 +129,19 @@ public void ValidateAppProtocolVersion()
                 messageTimestampBuffer: null,
                 differentAppProtocolVersionEncountered: callback);
             messageValidator.ValidateAppProtocolVersion(trustedPing1);
-            exception = Assert.Throws<DifferentAppProtocolVersionException>(
-                () => messageValidator.ValidateAppProtocolVersion(unknownPing1));
-            Assert.True(exception.Trusted);
+            Assert.False(called);
```

Comment:
I'm not sure should need this assert 😶

Reply:
I think it's fine for now as a part of the spec until there is some need to inspect messages from unknown signer from an application. 😶